db/seeds.rb

```diff
@@ -1,7 +1,37 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+require 'nokogiri'
+
+module JokeParser
+  def self.setup
+    seed_laffy_taffy
+    seed_chartcons
+  end
+
+  def self.seed_laffy_taffy
+    doc = Nokogiri::HTML(File.open('db/sources/laffy_taffy.html'))
+
+    doc.search('ol > li').map do |element|
+      joke = element.inner_text.strip.gsub(/^\d+\W\s/, "").downcase.capitalize.split("?")
+      question = "#{joke.first}?"
+      answer = joke.last.strip.gsub(/^\W */, "").capitalize
+      Joke.create!(question: question, answer: answer)
+    end
+    puts "Parsed Laffy Taffy joke file"
+  end
+
+  def self.seed_chartcons
+    doc = Nokogiri::HTML(File.open('db/sources/chartcons-jokes.html'))
+
+    doc.search('.wp-content > p').each_with_index do |element, index|
+      next if index == 0
+      next if element.first_element_child != nil
+      joke = element.inner_text.strip.split("?")
+      question = joke.first.gsub(/^\d+\W\s/, "")
+      answer = joke.last.strip.gsub(/^\W */, "")
```

Comment:
Do we want to capitalize these as well?

Also, totally minor thing, but do we want to mirror the pattern above that gives us `Joke.create!(question: question, answer: answer)`? Think it would be a minor switch to add the `?` to line 28. Like the way we do a simple split here and then the `gsub` separately, maybe take that approach above? Obviously this is super nit-picky and I'm fine either way. 👍